### DISCUSSION TITLE
Restrict multi monitor access

### DISCUSSION THREAD

**Discussion by friedPotat0** [rustdesk/rustdesk#3987] [Ideas]
It would be nice if there was a way to restrict access to one specific monitor. TeamViewer also has a meeting mode where you share a specific monitor and the other users can only see this particular monitor.

**Comment by rustdesk** [comment]
Thanks, Good feature. Technically it is easy.

**Comment by ic3man5** [comment]
I can use this feature, or at least a hotkey to switch between them.

**Comment by paboman** [comment]
very important feature for a remote desktop software imho
